Sources/Model/Receipt/HyperwalletReceipt.swift

```diff
@@ -127,109 +125,204 @@ public struct HyperwalletReceipt: Decodable, Equatable {
+        /// The donation return
         case donationReturn = "DONATION_RETURN"
-        // Merchant payment types
+        /// Merchant payment types
+        ///
+        /// The merchant payment
         case merchantPayment = "MERCHANT_PAYMENT"
+        /// The merchant payment fee
         case merchantPaymentFee = "MERCHANT_PAYMENT_FEE"
+        /// The merchant payment refund
         case merchantPaymentRefund = "MERCHANT_PAYMENT_REFUND"
+        /// The merchant payment return
         case merchantPaymentReturn = "MERCHANT_PAYMENT_RETURN"
-        // MoneyGram types
+        /// MoneyGram types
+        ///
+        /// The money gram transfer return
         case moneygramTransferReturn = "MONEYGRAM_TRANSFER_RETURN"
+        /// The transfer to money gram
         case transferToMoneygram = "TRANSFER_TO_MONEYGRAM"
-        // Paper check types
+        /// Paper check types
+        ///
+        /// The paper check fee
         case paperCheckFee = "PAPER_CHECK_FEE"
+        /// The paper check refund
         case paperCheckRefund = "PAPER_CHECK_REFUND"
+        /// The transfer to paper check
         case transferToPaperCheck = "TRANSFER_TO_PAPER_CHECK"
-        // User and program account types
+        /// User and program account types
+        ///
+        /// The account to be closed
         case accountClosure = "ACCOUNT_CLOSURE"
+        /// The account closure fee
         case accountClosureFee = "ACCOUNT_CLOSURE_FEE"
+        /// The account unload
         case accountUnload = "ACCOUNT_UNLOAD"
+        /// The dormant user fee
         case dormantUserFee = "DORMANT_USER_FEE"
+        /// The dormant user fee refund
         case dormantUserFeeRefund = "DORMANT_USER_FEE_REFUND"
+        /// The payment
         case payment = "PAYMENT"
+        /// The payment cancellation
         case paymentCancellation = "PAYMENT_CANCELLATION"
+        /// The payment reversal
         case paymentReversal = "PAYMENT_REVERSAL"
+        /// The payment reversal fee
         case paymentReversalFee = "PAYMENT_REVERSAL_FEE"
+        /// The payment return
         case paymentReturn = "PAYMENT_RETURN"
+        /// The transfer to program account
         case transferToProgramAccount = "TRANSFER_TO_PROGRAM_ACCOUNT"
+        /// The transfer to user
         case transferToUser = "TRANSFER_TO_USER"
-        // Virtual incentive types
+        /// Virtual incentive types
+        ///
+        /// The virtual incentive cancellation
         case virtualIncentiveCancellation = "VIRTUAL_INCENTIVE_CANCELLATION"
+        /// The virtual incentive issuance
         case virtualIncentiveIssuance = "VIRTUAL_INCENTIVE_ISSUANCE"
+        /// The virtual incentive purchase
         case virtualIncentivePurchase = "VIRTUAL_INCENTIVE_PURCHASE"
+        /// The virtual incentive refund
         case virtualIncentiveRefund = "VIRTUAL_INCENTIVE_REFUND"
-        // Western Union and WUBS types
+        /// Western Union and WUBS types
+        ///
```

Comment:
No need to add this in document so just // are fine and remove the extra ///

Reply:
Removed extra /